drivers/linux/chipsec_km.c

```diff
@@ -544,6 +544,13 @@ int __weak phys_mem_access_prot_allowed(struct file *file,
         return 1;
 }
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,15,4)
+int valid_mmap_phys_addr_range(unsigned long pfn, size_t size)
```

Comment:
Function signature is different than the original, is this on purpose?

Reply:
asm/io.h defines the function as follows:
#define ARCH_HAS_VALID_PHYS_ADDR_RANGE
extern u64 kern_mem_attribute (unsigned long phys_addr, unsigned long size);
extern int valid_phys_addr_range (phys_addr_t addr, size_t count); /* efi.c */
extern int valid_mmap_phys_addr_range (unsigned long pfn, size_t count);

When defined as static inline there were compilation errors due to mismatched function types.

Reply:
Thanks for checking that.